### DISCUSSION TITLE
v2.12.4

### DISCUSSION THREAD

**Discussion by jgraichen** [jgraichen/redmine_dashboard#351] [Announcements]
### Changes

- Improve Gemfile for broken docker container environments by @jgraichen

##  Looking for testers!

Unfortunately, I do not have access to any larger Redmine project anymore, nor am I using Redmine anywhere anymore. I can only test with some artificially constructed local project that cannot represent any real project or usage. I would really appreciate if you can test v2.12.4 in your project or organization and report any findings (or their absence). Thank you!

<hr /><em>This discussion was created from the release <a href='https://github.com/jgraichen/redmine_dashboard/releases/tag/v2.12.4'>v2.12.4</a>.</em>

**Comment by wnyn0818** [comment]
I can't start after installing on Docker, below is my log.
Redmine: 4.1.1
<img width="1784" alt="螢幕擷取畫面 2023-02-14 144714" src="https://user-images.githubusercontent.com/89724944/218660216-2488c884-af3a-429d-8e95-134fde1ece17.png">

**Reply by jgraichen** [reply]
@wnyn0818 This is not an issue with the plugin, but you do not have permissions to run `bundle install`. Maybe you are running as the wrong user inside docker. Please consult the image provider.

Note that Ruby 2.6 and Redmine 4.1 are very old, end of life, and not maintained anymore.
